src/tools/web/README.md

```diff
@@ -114,6 +114,53 @@ Elektra web consists of multiple components:
 - [elektrad](https://master.libelektra.org/doc/api_blueprints/elektrad.apib), documentation: https://elektrad.docs.apiary.io/
 - [webd](https://master.libelektra.org/doc/api_blueprints/webd.apib), documentation: https://elektrawebd.docs.apiary.io/
 
+## Test REST API on localhost
+
+In order to test API on localhost, your have to start elektrad instance. You can do it in two ways:
+
+- run manually
+
+  - `cd libelektra/src/tools/web`
+  - `cd elektrad`
+  - `npm install`
+  - `npm start` (replaces `kdb run-elektrad`)
```

Comment:
What means "replaces" here? From source you use `npm start`, if installed `kdb run-elektrad`. It seems to be that this instructions duplicate the instructions above or web/elektrad?

Reply:
Actually, I just copy pasted what is written in https://www.libelektra.org/tools/web in the section "Running from source".

It means, that all 4 commands:
- `cd libelektra/src/tools/web`
- `cd elektrad`
- `npm install`
- `npm start`

replace kdb run-elektrad

Reply:
> Actually, I just copy pasted

Please don't do this.

https://www.libelektra.org/tools/web is generated from this document.